awx/main/tasks.py

```diff
@@ -1596,6 +1649,13 @@ def get_password_prompts(self, passwords={}):
                 d[r'Vault password \({}\):\s*?$'.format(vault_id)] = k
         return d
 
+    def should_use_resource_profiling(self, job):
+        '''
+        Return whether this task should use resource profiling
+        '''
+        return getattr(settings, 'AWX_RESOURCE_PROFILING_ENABLED', False) or \
+            getattr(job, 'verbosity', 0) >= 5
```

Comment:
As discussed, let's remove this verbosity enablement behavior.

Reply:
yep, have been working on iso node verification (took a while to figure out that it was a runner packaging issue, not a code issue) and on the second round of benchmark testing. Will take care of this in the a.m.

Reply:
removed